From 0e5c456e8ee436d835624cc2578e7064247ed033 Mon Sep 17 00:00:00 2001
From: Yufei Gu <yufei.gu@cloudera.com>
Date: Thu, 12 Jan 2017 15:02:38 -0800
Subject: [PATCH 2147/2848] MAPREDUCE-5981. Log levels of certain MR logs can
 be changed to DEBUG. Contributed by Varun Saxena.

(cherry picked from commit dc2b2ae31f2eb6dae324c2e14ed7660ce605a89b)
(cherry picked from commit a957baeb71eb75aa259902d9d6dce2a64f5a3e38)

Change-Id: If4f26ffaf6dcc0cb8cb0090ff791bd93a2c896e2
---
 .../hadoop/mapreduce/task/reduce/Fetcher.java      |    2 +-
 .../task/reduce/ShuffleSchedulerImpl.java          |    4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/task/reduce/Fetcher.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/task/reduce/Fetcher.java
index 5d7bb11..0485228 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/task/reduce/Fetcher.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/task/reduce/Fetcher.java
@@ -473,7 +473,7 @@ private void verifyConnection(URL url, String msgToEncode, String encHash)
     LOG.debug("url="+msgToEncode+";encHash="+encHash+";replyHash="+replyHash);
     // verify that replyHash is HMac of encHash
     SecureShuffleUtils.verifyReply(replyHash, encHash, shuffleSecretKey);
-    LOG.info("for url="+msgToEncode+" sent hash and received reply");
+    LOG.debug("for url="+msgToEncode+" sent hash and received reply");
   }
 
   private void setupShuffleConnection(String encHash) {
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/task/reduce/ShuffleSchedulerImpl.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/task/reduce/ShuffleSchedulerImpl.java
index 7489e63..a819771 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/task/reduce/ShuffleSchedulerImpl.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/task/reduce/ShuffleSchedulerImpl.java
@@ -447,7 +447,7 @@ public synchronized MapHost getHost() throws InterruptedException {
       pendingHosts.remove(host);
       host.markBusy();
 
-      LOG.info("Assigning " + host + " with " + host.getNumKnownMapOutputs() +
+      LOG.debug("Assigning " + host + " with " + host.getNumKnownMapOutputs() +
                " to " + Thread.currentThread().getName());
       SHUFFLE_START.set(Time.monotonicNow());
 
@@ -477,7 +477,7 @@ public synchronized MapHost getHost() throws InterruptedException {
         host.addKnownMap(id);
       }
     }
-    LOG.info("assigned " + includedMaps + " of " + totalSize + " to " +
+    LOG.debug("assigned " + includedMaps + " of " + totalSize + " to " +
              host + " to " + Thread.currentThread().getName());
     return result;
   }
-- 
1.7.9.5

